src/js/controller/instream-adapter.js

```diff
@@ -164,8 +164,10 @@ define([
             }
 
             if (_array && _arrayIndex + 1 < _array.length) {
-                // fire complete event
-                this.trigger(events.JWPLAYER_MEDIA_COMPLETE, data);
+                // fire complete event if not ad skipped
+                if (e.type !== events.JWPLAYER_AD_SKIPPED) {
+                    this.trigger(events.JWPLAYER_MEDIA_COMPLETE, data);
+                }
```

Comment:
@pajong this shouldn't be necessary. This points to a problem with the ad plugins and instream that you missed:

`_instreamItemNext` is fired on mediaComplete. Skipping an add should not trigger mediaComplete.
```
_instream.on(events.JWPLAYER_MEDIA_COMPLETE, _instreamItemNext, this);
```

Did it at some point? Because it looks like you already tried to work around this with `setupSkipButton` `customNext`:
```js
        this.setupSkipButton = function(skipoffset, options, customNext) {
            _model.set('skipButton', false);
            if (customNext) {
                _instreamItemNext = customNext;
            }
            _instream._adModel.set('skipMessage', options.skipMessage);
            _instream._adModel.set('skipText', options.skipText);
            _instream._adModel.set('skipOffset', skipoffset);
            _model.set('skipButton', true);
        };
```